routes/ama.js

```diff
@@ -88,6 +100,24 @@ router.post('/candidate/:rcs_id', function (req, res) {
 
     connection.query(query, functions.defaultJSONCallback(res));
 
+    let mailoptions = {
+        text: "The following question has been asked on your profile: \n" +
+        data.question_text + "\n \n" +
```

Comment:
Ideally we'd link to the question here too, but we don't have a URL route for individual questions. So the best we could do is link to the candidate's AMA page, e.g. https://elections.union.rpi.edu/candidate/etzinj/ama.